### PR TITLE
[New Contributors of Commits] Re-implement the metric to make similar to definition

### PR DESCRIPTION
**Pull request by Polaris000**
This pull request modifies the current python implementation, making it according to the definition of the metric by the Working Group.

Additionally, a copy issue is fixed.

### CODE REVIEW

File: implementations/code_df/new_contributors_of_commits_git.py [hunk 1]
```diff
@@ -81,39 +144,56 @@ def _agg(self, df, period):
...
     new_committers_interval = NewContributorsOfCommitsGit(
         items,
         (date_since, date_until),
-        is_code=[conditions.DirExclude(['tests']),
-                 conditions.PostfixExclude(['.md', 'COPYING'])])
+        is_code=[DirExclude(['tests']),
+                 PostfixExclude(['.md', 'COPYING'])])
 
-    # time-series on a monthly basis
+    # # time-series on a monthly basis
```
Comment (valeriocos) [inline implementations/code_df/new_contributors_of_commits_git.py]: Typo, double `#`

Comment (Polaris000) [reply implementations/code_df/new_contributors_of_commits_git.py]: Oops! Will fix this. Thanks!

**Comment by valeriocos** [comment]
@jgbarah we can merge it and maybe reiterate on the code with another pr

**Comment by jgbarah** [comment]
Yes, i agree. Thanks.

**Pull request merged**